src/App.css

```diff
@@ -0,0 +1,312 @@
+.add-title {
+  color: #888;
+  width: 10.875rem;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```

Comment:
- [ ] Kindly refer to the first comment for these three CSS declarations repeated many times. 👍